Auto case OCP-22665

kasturinarra opened the pull request:
Auto case to check etcd image have been update to target release value after upgrade

kasturinarra commented:
@geliu2016 could you please help review ? Thanks !!

https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/2232/console

pruan-rht commented on features/upgrade/etcd/upgrade.feature:
```diff
@@ -29,4 +29,31 @@ Feature: basic verification for upgrade testing
+  @admin
+  Scenario: Check etcd image have been udpated to target release value after upgrade
+    # operands
+    When I run the :get admin command with:
```
@kasturinarra we recommend using the object methods instead of calling the raw `:get`.  You can replace 39-44 with ```    Given I switch to cluster admin pseudo user
    And I use the "openshift-etcd" project
    And a pod becomes ready with labels:
      | app=etcd,etcd=true |
    And evaluation of `pod.container_specs.first.image` is stored in the :etcd_image clipboard
```

kasturinarra replied:
addressed  !!

pruan-rht commented on features/upgrade/etcd/upgrade.feature:
```diff
@@ -29,4 +29,31 @@ Feature: basic verification for upgrade testing
+      | o        | jsonpath={.items[0].spec.containers[0].image} |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :etcd_image clipboard
+    When I run the :get admin command with:
```
@kasturinarra same applies to 45-49.  You can replace it with 
```
And evaluation of `cluster_version('version').image` is stored in the :payload_image clipboard
```

kasturinarra replied:
addressed !!

kasturinarra commented:
@pruan-rht i have addressed all your comments, could you please help check ? Thanks !!

https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/2277/console

pruan-rht reviewed (commented):
/lgtm
